src/ActionsImporter/Commands/ListFeatures.cs

```diff
@@ -12,5 +12,7 @@ public ListFeatures(string[] args) : base(args)
     protected override string Name => "list-features";
```

Comment:
I'm on the fence about this.

I do see the benefit of simplifying the command, though this command is explicitly _listing_ the features. And I'd probably want to change `github/valet` to match as well (it currently uses `list-features`) as well.

And part of the reason I went with `list-features` was to avoid some confusion around the `--features` option that exists on global commands.

Given that context, do you still feel strongly about changing this name?

Reply:
Ah, I forgot about the `--features` option that we had already. I'm ok with leaving this as is then 👍